tv/interfaces/room/index.ts

```diff
@@ -0,0 +1,21 @@
+export interface Result {
```

Comment:
`Result` seems a bit generic for a name. Other interfaces files have something like `RoomResult` for that interface. Could it be renamed?